ciscript

```diff
@@ -47,4 +47,5 @@ llvm-kompile ../test/test9.kore TEST main -o interpreter
 ./interpreter ../test/test9.kore -1 /dev/stdout | diff - ../test/test9.out
 llvm-kompile ../test/test10.kore TEST main -o interpreter
 ./interpreter ../test/test10.in.kore -1 /dev/stdout | diff - ../test/test10.out
+llvm-kompile ../test/test11.kore TEST main -o interpreter
```

Comment:
What about running the generated interpreter?

Reply:
This test just tests that the definition compiles. I wrote the more detailed tests to be part of the k frontend and I believe we should pursue this path moving forward as the testing infrastructure there will be more robust and stable.